### ISSUE TITLE
Is env.parameters.NB_TIMESTEP_COOLDOWN_SUB 0 or 3?

### ISSUE THREAD

**Issue by richardwth** [rte-france/Grid2Op#366]
`env.parameters.NB_TIMESTEP_COOLDOWN_SUB` is 0 in training data but 3 in starting_kit validation data. Should it be 0 or 3 in online test?

```
from grid2op import make

env = make("l2rpn_wcci_2022")
print(env.parameters.NB_TIMESTEP_COOLDOWN_SUB)  # This gives 0

env = make("../L2RPN_wcci2022_starting_kit/input_data_local")
print(env.parameters.NB_TIMESTEP_COOLDOWN_SUB)  # This gives 3
```

**Comment by BDonnot** [comment]
Hello,

I'll check asap, but it should be 3.

Benjamin

**Comment by richardwth** [comment]
I further checked other parameters...and found four differences

`make("l2rpn_wcci_2022").parameters.__dict__` gives
```
'NB_TIMESTEP_OVERFLOW_ALLOWED': 2,
'NB_TIMESTEP_RECONNECTION': 10,
'NB_TIMESTEP_COOLDOWN_LINE': 0,
'NB_TIMESTEP_COOLDOWN_SUB': 0,
```
`make("../L2RPN_wcci2022_starting_kit/input_data_local").parameters.__dict__` gives
```
'NB_TIMESTEP_OVERFLOW_ALLOWED': 3,
'NB_TIMESTEP_RECONNECTION': 12,
'NB_TIMESTEP_COOLDOWN_LINE': 3,
'NB_TIMESTEP_COOLDOWN_SUB': 3,
```
I think this should be highlighted somewhere, as one typically trains models on `env=make("l2rpn_wcci_2022")` but the training setting and test setting are different.

**Comment by BDonnot** [comment]
Hello,

Sorry for late reply. You are totally correct, we missed to check the environment parameters for the training set (and only checked the one in the warmup, validation and test phases).

Sorry for this error, I will re upload the proper data ASAP. We forgot to include the "difficulty_levels.json" file in the training environment. Sorry for that.

Benjamin

**Comment by BDonnot** [comment]
I close the issue as the competition is over :-)

Congrats on your rank by the way !
